plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -146,6 +146,15 @@ public void testSessionCatalogSchema()
         assertUpdate(session, "DROP MATERIALIZED VIEW iceberg.tpch.materialized_view_session_test");
     }
 
+    @Test
+    public void testDropIfExists()
```

Comment:
add a test with Hive too.

utilmately, we shouild have `BaseConnectorTest` coverage for materialized views. cc @sopel39 @losipiuk

Reply:
Hive connector doesn’t support MVs.
I'll look into adding BaseConnectorTest coverage for materialized views in future.

Reply:
Thanks, let's maybe have an issue for `BaseConnectorTest` coverage.

Reply:
Created https://github.com/trinodb/trino/issues/8054